plugins/de.cognicrypt.staticanalyzer/META-INF/MANIFEST.MF

```diff
@@ -23,4 +23,5 @@ Bundle-ActivationPolicy: lazy
 Bundle-ClassPath: .,
  resources/
 Automatic-Module-Name: de.cognicrypt.staticanalyzer
-Import-Package: org.eclipse.ui.texteditor
+Import-Package: de.cognicrypt.staticanalyzer.kotlin.utilities,

```

Comment:
If there is a dependency between these two plugins, it should be the other way around. Please check why this dependency exists and resolve it differently.